dashboard/views.py

```diff
@@ -330,13 +328,13 @@ def resource_access_within_week(request, course_id=0):
 
     output_df['resource_name'] = output_df.apply(
         lambda row:
-            RESOURCE_URLS[row.resource_type]["prefix"] +
+            RESOURCE_ACCESS_CONFIG.get(row.resource_type).get("urls").get("prefix") +
             row.resource_id_part +
-            RESOURCE_URLS[row.resource_type]["postfix"] +
+            RESOURCE_ACCESS_CONFIG.get(row.resource_type).get("urls").get("postfix") +
             CANVAS_FILE_ID_NAME_SEPARATOR +
             row.resource_name_part + CANVAS_FILE_ID_NAME_SEPARATOR +
-            RESOURCE_VALUES[INVERSE[row.resource_type]]['icon'],
-        axis=1)
+            RESOURCE_VALUES.get(RESOURCE_VALUES_MAP.get(row.resource_type)).get('icon'),
+            axis=1)
```

Comment:
The `axis` is actually a parameter for `.apply()`. Seems like it should be un-indented. If that's confusing, maybe we should put parentheses around the concatenated strings.

Reply:
Yeah, this is a little confusing with the lambda where, let's do that.